integration/experiment/gpu_frequency_sweep/gen_gpu_activity_constconfig_recommendation.py

```diff
@@ -49,7 +50,7 @@ def extract_columns(df):
 
     return df_cols
 
-def policy_efficient_energy(df):
+def efficient_energy(df):
```

Comment:
The function name makes it seem like it will return some amount of energy, but it actually returns a frequency.